cmd/reviewdog/main.go

```diff
@@ -317,21 +317,57 @@ func githubService(ctx context.Context) (gs *githubservice.GitHubPullRequest, is
 	if err != nil {
 		return nil, isPR, err
 	}
-	// TODO: support commit build
-	if !isPR {
-		return nil, isPR, nil
-	}
 
 	client, err := githubClient(ctx, token)
 	if err != nil {
 		return nil, isPR, err
 	}
 
+	if !isPR {
+		prID, err := getPullRequestIDByBranchOrCommit(ctx, client, g)
+		if err != nil {
```

Comment:
log error and `return  nil, false, nil` instead of returning error.

Reply:
Done.

Reply:
I wonder how this work when REVIEWDOG_GUESS_PULL_REQUEST is set and failed to get pull-request-id. Exit with code 0? or non-0?

Reply:
In this case, reviewdog will exit with code 0.